app/keepers/keepers.go

```diff
@@ -116,9 +126,18 @@ type AppKeepers struct {
 	ICAHostKeeper       *icahostkeeper.Keeper
 	FeeMarketKeeper     *feemarketkeeper.Keeper
 
+	PFMRouterKeeper *packetforwardkeeper.Keeper
+	RatelimitKeeper ratelimitkeeper.Keeper
+
 	// IBC modules
 	// transfer module
 	RawIcs20TransferAppModule transfer.AppModule
+	RateLimitModule           ratelimit.AppModule
+	TransferStack             *ibchooks.IBCMiddleware
+	TransferModule            transfer.AppModule
```

Comment:
Why do we add transfer module here when we have RawIcs20TransferModule already?

Reply:
removed `RawIcs20TransferAppModule`